services/workflows-service/src/common/form-data-validation.pipe.ts

```
@@ -0,0 +1,48 @@
+
+    if (errors?.length) {
+      const errorMessages = errors
+        .map(error => Object.values(error.constraints || {}).join(', '))
+        .join('; ');
+      throw new BadRequestException(errorMessages);
+    }
+
+    return object;
+  }
+
+  private toValidate(metatype: Function): boolean {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)